config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,117 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Is this file needed?

Reply:
Yeah I would like to keep it there until next rails update.